contrib/misc/testlaggedcoherence.m

```diff
@@ -0,0 +1,12 @@
+datain=[];
```

Comment:
This function is a duplicate of the function in fieldtrip/test, called test_laggedcoherence.m Perhaps best to make the changes there, and remove this file from the contrib/misc/ folder

Reply:
OK. Hoe had ik dit kunnen weten?

Reply:
niet. 

But it seems that the equivalent fieldtrip/test/test_laggedcoherence looks uncannily similar to the testlaggedcoherence in contrib/misc. So, perhaps I already moved it over at some point.